src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+    rel="stylesheet"
+    >
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body class="page"> 
+    <div class="card" data-qa="card">
+      <img 
+        src="images/imac.jpeg" 
+        alt="img" 
+        class="card__image card__image--position"
+      >
+      <h2 class="card__name card__name--position">
```

Comment:
Do you really need modifier `card__name--position`?
As style.css is empty now? i can't see for what it.